cmd/stack/deploy.go

```diff
@@ -99,7 +99,9 @@ func (c *DeployCommand) RunDeploy(ctx context.Context, s *model.Stack, options *
 			if err != nil {
 				return err
 			}
-			nsCmd.Create(ctx, &namespace.CreateOptions{Namespace: s.Namespace})
+			if err := nsCmd.Create(ctx, &namespace.CreateOptions{Namespace: s.Namespace}); err != nil {
```

Comment:
We should be returning an error here, we will fail later on because the ns doesn't exist